blockdag/validate_test.go

```diff
@@ -194,6 +195,30 @@ func TestCheckBlockSanity(t *testing.T) {
 		t.Errorf("CheckBlockSanity: unexpected return %s delay", delay)
 	}
 
+	// Ensure that a block that's bigger in bytes than MaxMassPerBlock fails
+	blockBytes, err := block.Bytes()
+	if err != nil {
+		t.Fatalf("CheckBlockSanity: Bytes unexpectedly failed: %s", err)
+	}
+	tooBigMassBlock, err := util.NewBlockFromBytes(blockBytes)
```

Comment:
Why call it tooBigMass? It's tooBigBytes, no? Or you can just call it tooBigBlock?